$MirOS: ports/emulators/qemu/patches/patch-audio_ossaudio_c,v 1.2 2005/05/03 19:30:03 tg Exp $
--- audio/ossaudio.c.orig	Wed Apr 27 20:51:43 2005
+++ audio/ossaudio.c	Tue May  3 18:01:48 2005
@@ -21,10 +21,15 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+#ifdef __OpenBSD__
+#include <stdlib.h>
+#include <soundcard.h>
+#else
+#include <sys/soundcard.h>
+#endif
 #include <sys/mman.h>
 #include <sys/types.h>
 #include <sys/ioctl.h>
-#include <sys/soundcard.h>
 #include <assert.h>
 #include "vl.h"
 
